tests/compute/test_transform.py

```diff
@@ -1480,5 +1480,86 @@ def test_remove_selfloop(idtype):
         raise_error = True
     assert raise_error
 
+@parametrize_dtype
+def test_reorder(idtype):
```

Comment:
This test is not added to `__main__` yet.

Reply:
Talked to Minjie. we test with pytest for these tests.

ubuntu@ip-172-31-24-140:~/dgl1/tests$ pytest-3 compute/test_transform.py::test_reorder
=================================== test session starts ===================================
platform linux -- Python 3.6.9, pytest-3.3.2, py-1.5.2, pluggy-0.6.0
rootdir: /home/ubuntu/dgl1/tests, inifile:
collected 2 items                                                                         

compute/test_transform.py ..                                                        [100%]

================================ 2 passed in 0.92 seconds =================================